### PR TITLE
fix: run command validating inputs caused inputs to not work in crawlee projects

### LINKED ISSUE #586: Actor.getInput returns null
After the latest release, the `await Actor.getInput()` functionality got broken. It returns `null` and cannot retrieve the input field properly.

`apify run -p` command breaks the `getInput` function. `npm run src/main.js`returns properly. Possibly some lib clash is happening.

**Apify CLI version:** `apify-cli/0.20.0 darwin-arm64 node-v18.17.1`
`apify` lib version: `3.2.3`
`crawlee` version: `3.5.4`

Also, I tried scaffolding the project by using the `apify` CLI tool but can reproduce the error still.

### PR DESCRIPTION
**Pull request by vladfrangu**
Closes #586

### CODE REVIEW

**Review by B4nan — COMMENTED** [review]
that's a lot of changes, i would like to see some tests too

**Review by B4nan — APPROVED** [review]
just one nit

File: src/commands/run.ts [hunk 1]
```diff
@@ -370,43 +340,67 @@ export class RunCommand extends ApifyCommand<typeof RunCommand> {
...
+        const existingInput = getLocalInput(process.cwd());
+
+        // Prepare the file path for where we'll temporarily store the validated input
+        const inputFilePath = join(process.cwd(), getLocalKeyValueStorePath(), existingInput?.fileName ?? 'INPUT.json');
+
+        let errorHeader: string;
+
+        switch (inputOverride?.source) {
+            case 'stdin':
+                errorHeader = 'The input provided through standard input is invalid. Please fix the following errors:\n';
```
[the remark this answers is not in the document]
Comment (vladfrangu) [reply src/commands/run.ts:353]: Woops, nice catch

[the remark this answers is not in the document]
Comment (vladfrangu) [reply src/commands/run.ts:353]: Actually, in this case switching it around sounds wrong, does it not?

[the remark this answers is not in the document]
Comment (B4nan) [reply src/commands/run.ts:353]: Right, I just saw the first few words and didn't read the whole sentence :] Let's keep it like this, its a minor thing anyway.

**Pull request merged**